static/templates/settings/organization-profile-admin.handlebars

```diff
@@ -51,11 +51,20 @@
         </a>
 
         <div class="subsection-header">
-            <h3>{{t "Organization logo" }}
-                <a href="/help/create-your-organization-profile#add-a-wide-logo" target="_blank">
-                    <i class="fa fa-question-circle-o" aria-hidden="true"></i>
-                </a>
-            </h3>
+            <span>
+                <h3>{{t "Organization logo" }}
+                    <a href="/help/create-your-organization-profile#add-a-wide-logo" target="_blank">
+                        <i class="fa fa-question-circle-o" aria-hidden="true"></i>
+                    </a>
+                </h3>
+            </span>
+            <span>
+                <h5>
+                    {{#unless plan_includes_wide_organization_logo}}
+                    {{t upgrade_text_for_wide_organization_logo}} <a href="https://zulipchat.com/upgrade/"> <i class="fa fa-rocket" aria-hidden="true"></i> </a>
```

Comment:
Instead of using `<h5>` and adding the icon in the html, let's style the whole thing using a CSS class. I think we can potentially even add the rocket with a `:before` type thing?

Reply:
I did this, and for the time being (until Tim Abbott's review), I've changed the hyperlink to cover the entire string (now excluding the rocket because of a `::after` clause). This is just a placeholder until we decide where we want the hyperlink to go and how.

Reply:
@timabbott, are we sure that the way we hyperlinked the whole string is a good idea (speaking from the perspective of aesthetics)?